main/glossary/README.md

```diff
@@ -318,34 +324,34 @@ records with values of [amounts](#amounts), [issuers](#issuer), etc.
 ## Mint
 
 [ERTP](#ertp) has a *mint* object, which creates digital assets. [ZCF](#zcf) provides a different interface to an ERTP mint, called a
-*ZCFMint*. Assets and AssetHolders created using ZcfMints can be used in all the same ways as assets created by other ERTP Mints.
+*ZCFMint*. Assets and AssetHolders created using ZCFMints can be used in all the same ways as assets created by other ERTP Mints.
 They interact with Purses, Payments, Brands, and Issuers in the same ways.
 
 - ERTP mints create digital assets and are the only ERTP objects with the authority to do so.
   Access to an ERTP mint gives you the power to create more digital assets of its type at will. Mints
   can only create one type of asset and cannot change to create a different type.
 
-  ERTP mints are [issuer's](#issuer) admin [facets](#facet), and there is a one-to-one relationship between an issuer and
-  its mint. ERTP mints are also in a one-to-one relationship with that issuer's associated [brand](#brand).
+  There is a one-to-one relationship between an [issuer](#issuer) and its mint, and a mint is an administrative [facet](#facet) of its issuer.
+  ERTP mints are also in a one-to-one relationship with that issuer's associated [brand](#brand).
 
 - ZCFMints give contract code a simpler interface to interact with an ERTP mint. Because ZCFMints encapsulate
   an internal ERTP mint, they have the same one-to-one relationships
   with an issuer and its associated brand. A ZCFMint can mint assets and assign them to a seat without having to escrow
-  payments, and burn assets that used to be associated with a seat without having to payout assets.
+  payments, and burn seat-associated assets without corresponding payout.
 
 ZCFMints and ERTP mints do **not** have the same methods. Do not try to use ERTP methods on a ZCFMint or vice versa.
```

Comment:
I can't decide whether it is helpful to point out that a ZCFMint actually wraps an ERTP Mint. Do whatever seems best.